recastjs/recast.d.ts

```diff
@@ -111,4 +113,11 @@ declare module "recast-detour" {
     getDefaultQueryExtent(): Vec3;
     getCorners(idx: number): NavPath;
   }
-}
\ No newline at end of file
+  export interface RecastDetourModule extends EmscriptenModule {
```

Comment:
I believe this interface should hold the entier implementation (the classes, objects exported, etc').

Reply:
Have you tested this declaration in a project? I can do it later

Reply:
I builds with the test project